uiowa/uiowa
# Featured content block type

Relates to #2259 

# How to test
- `blt ds --site sandbox.uiowa.edu`
- `drush @sandbox.local uli /`
- Add a featured content block and confirm the following stories can be achieved:
  - As a site owner, I add a featured content block to a Page.
  - As a site owner, I can update the featured content block to add new featured items.
  - As a site owner, I can remove items from the featured content block.
  - As a site owner, I can reorder the items in the featured content block.
  - Repeat the following for all of these sites/features... or visually review:
    - apr feature
    - areas of study feature
    - event feature
    - scholarship feature
    - admissions
    - grad
    - policy.clas
    - presidentialsearch
    - transportation
    - uiowa.edu
